custom/enikshay/integrations/bets/repeaters.py

```diff
@@ -51,12 +51,146 @@ def allowed_to_forward(self, voucher_case):
         return (
             approved
             and not_sent
+            # TODO: Check if voucher is related to test location?
             and case_properties_changed(voucher_case, ['state'])
         )
 
 
+class BETS180TreatmentRepeater(BaseBETSRepeater):
+    friendly_name = _("BETS - MBBS+ Providers: 180 days of private OR govt. FDCs with treatment outcome reported (episode case type)")
+
+    @classmethod
+    def get_custom_url(cls, domain):
+        from custom.enikshay.integrations.bets.views import BETS180TreatmentRepeaterView
+        return reverse(BETS180TreatmentRepeaterView.urlname, args=[domain])
+
+    def allowed_to_forward(self, episode_case):
+        if not self.case_types_and_users_allowed(episode_case):
+            return False
+
+        case_properties = episode_case.dynamic_case_properties()
+        treatment_outcome = case_properties.get('treatment_outcome', None)
+        episode_has_outcome = treatment_outcome and (treatment_outcome != 'not_evaluated')
```

Comment:
I think you should also check that this is "transitioned" to an outcome with `case_properties_changed`. Otherwise we could get into a situation where we send this multiple times for every form submission against this case until the `event_{}` gets set to sent.